.circleci/config.yml

```diff
@@ -1,16 +1,41 @@
 version: 2
 jobs:
-  build:
-    docker:
-      - image: rocker/tidyverse:3.6.0
-
+  run_tests:
```

Comment:
Can you make this `run_analyses` so that it's easier to for external contributors to get into what's happening here?